src/applications/vaos/appointment-list/components/cards/confirmed/AppointmentDateTime.jsx

```diff
@@ -8,21 +8,35 @@ export default function AppointmentDateTime({
   appointmentDate,
   timezone,
   facilityId,
+  newFormat,
```

Comment:
What do you think about naming this something that's more descriptive about what the prop does? Sort of worried that `newFormat` will be confusing in the future. Maybe `videoFormat` or `displayOnTwoLines` or something to that effect?

Reply:
Sure. I was planing to change the name after doing the home appointment. I would imagine another refactor should be done once all of the separate changes have been merged into master.

Reply:
Ah, ok. I can update as part of my PR, since I think I have the last two video updates.